### DISCUSSION TITLE
Embeddings autofill?

### DISCUSSION THREAD

**Discussion by GrubbyGroo** [vladmandic/sdnext#304] [Q&A] [answered]
Sorry if this has been asked, but i couldn't find it. Is there a way to get embeddings to autofill?

Loras looks to work by doing <lora, but I can't do the same with embeddings like EastNegative

**Comment by vladmandic** [comment]
you mean to populate prompt? 
for lora, its easy since its just a filename.
for embedding, it cant be done since embedding filename is irrelevant. embeddings are are loaded immediately when model is loaded and each embedding extends built-in dictionary by one or multiple words. so what you're asking for would be "can i see a list of all currently known words in dictionary that are not present in normal dictionary".

**Comment by hollowstrawberry** [comment, marked as answer]
> Loras looks to work by doing <lora, but I can't do the same with embeddings like EastNegative

If you have the TagComplete extension, embeddings also come up when you type `<`, eg `<eas` for easynegative

**Reply by GrubbyGroo** [reply]
Thank you! I'll check and make sure that is added

**Comment by Javicshz** [comment]
You can save Prompts as Styles, this will auto fill prompts.

1. Enter the prompts you want saved
2. Click the Save icon under the Generate button, enter the name you want ex: "Default Prompts"

Next time you want to use the default prompts click on the drop down menu below the generate button and select your style. Then click the clipboard icon next to the save button and it will autofill the prompts
